apispec/insolvency-delta-spec.yml

```diff
@@ -87,6 +87,7 @@ components:
           type: string
         appointments:
           type: array
+          default: []
```

Comment:
I think this change need to be in api spec of private-api-specifications component, not in chs-delta-api